WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -115,7 +116,9 @@ private void StepOutputRegistrationPhase()
 				long aliceSum = round.Alices.Sum(x => x.CalculateRemainingAmountCredentials(round.FeeRate));
 				long bobSum = round.Bobs.Sum(x => x.CredentialAmount);
 				var diff = aliceSum - bobSum;
```

Comment:
This can be simplified, no need to sum over alices & bobs and convert

```cs
var diffMoney = round.CoinjoinState.Balance
```

(or `coinjoin.Balance` inside the if body)

Reply:
That is not the same, the fees were extracted from the sum.